cmd/chihaya/config.go

```diff
@@ -10,14 +10,16 @@ import (
 	"github.com/chihaya/chihaya/frontend/http"
 	"github.com/chihaya/chihaya/frontend/udp"
 	"github.com/chihaya/chihaya/middleware"
-
-	// Imported to register as middleware drivers.
+)
+import (
+	// Imports to register middleware drivers.
 	_ "github.com/chihaya/chihaya/middleware/clientapproval"
 	_ "github.com/chihaya/chihaya/middleware/jwt"
 	_ "github.com/chihaya/chihaya/middleware/torrentapproval"
 	_ "github.com/chihaya/chihaya/middleware/varinterval"
-
-	// Imported to register as storage drivers.
+)
+import (
```

Comment:
Does gofmt not add new lines between import blocks? If it's true, then LGTM else, I'd prefer them to be here, it looks strange w/o them.